ddl/job_table_test.go

```diff
@@ -200,6 +200,7 @@ func TestConcurrentDDLSwitch(t *testing.T) {
 
 	tk := testkit.NewTestKit(t, store)
 	tk.MustExec("use test")
+	tk.MustExec("set global tidb_enable_mdl=0")
```

Comment:
Is necessary to set to tidb_enable_mdl to 0?

Reply:
MDL doesn't support non concurrent DDL